_posts/2018-03-21-building-a-model-for-event-data-as-a-graph/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Building a model for event data as a graph"
+title-short: Event data as a graph
+tags: []
+author: Dilyan
+category:
+permalink: /blog/2018/03/21/building-a-model-for-atomic-event-data-as-a-graph/
+discourse: true
+---
+
+In recent months we've been busy expanding the variety of storage targets available for Snowplow users to load [Snowplow enriched events][enriched-event]. We recently launched our Snowflake Loader, and work is underway to add support for Google's BigQuery. Thinking even further ahead, one intriguing option is to add a graph-based storage target for Snowplow.
+
+We'd like to take the community with us on this journey, so we will be documenting our progress in a series of blog posts over the coming weeks and months. In this post we start by exploring options for modelling event data as a graph in general. Later on, we'll look into modelling specific events and contexts of relevant entities. We'll also look into options for storing and querying graph event data.
+
+In this post:
+
+1. [Why graph databases?](#why-graph-databases)
+2. [How do we model event data as a graph?](#how-do-we-model-event-data-as-a-graph)
+3. [Taking an event-grammar approach](#taking-an-event-grammar-approach)
+4. [Taking a time-series or event-graph approach](#taking-a-time-series-or-event-graph-approach)
+5. [Events as nodes versus events as edges](#events-as-nodes-versus-events-as-edges)
+6. [Do we only want to load atomic event data in a graph database?](#do-we-only-want-to-load-atomic-event-data-in-a graph-database)
+7. [What should be our pilot storage target?](#what-should-be-our-pilot-storage-target)
+
```

Comment:
You missed 8., the next post in the series

Reply:
@dilyand @alexanderdean I'll add it in- but otherwise it looks good on my end for posting.

Reply:
I left it out on purpose (just felt it didn't belong) but am agnostic about adding it. Thanks @Mandelliant !